config/webhook/manifests.yaml

```diff
@@ -10,9 +10,9 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /mutate-opentelemetry-io-v1alpha1-instrumentation
+      path: /mutate-opentelemetry-io-v1alpha1-opentelemetrycollector
```

Comment:
why are these changes here? 

I am concerned that this will cause issues for the upgrade (also the upgrade test was changed )